doc/source/whatsnew/v0.17.0.txt

```diff
@@ -599,6 +599,7 @@ Bug Fixes
 - Bug that caused segfault when resampling an empty Series (:issue:`10228`)
 - Bug in ``DatetimeIndex`` and ``PeriodIndex.value_counts`` resets name from its result, but retains in result's ``Index``. (:issue:`10150`)
 - Bug in ``pd.eval`` using ``numexpr`` engine coerces 1 element numpy array to scalar (:issue:`10546`)
+- Bug in ``PeriodIndex.__contains__`` that always returned False for datetime objects (:issue:`10798`)
```

Comment:
this is only for datetime.date only? right
I though datetime/Timestamps worked?